app/src/main/java/com/hous/hous_aos/data/repository/NewRulesRepositoryImpl.kt

```diff
@@ -1,28 +1,28 @@
 package com.hous.hous_aos.data.repository
 
 import android.util.Log
-import com.hous.hous_aos.data.model.WrapperClass
-import com.hous.hous_aos.data.model.request.Member
-import com.hous.hous_aos.data.model.request.NewRulesRequest
-import com.hous.hous_aos.data.model.response.NewRulesListResponse
-import com.hous.hous_aos.data.source.remote.RemoteNewRulesDataSource
+import com.hous.data.model.WrapperClass
+import com.hous.data.model.request.Member
+import com.hous.data.model.request.NewRulesRequest
+import com.hous.data.model.response.NewRulesListResponse
+import com.hous.data.source.remote.RemoteNewRulesDataSource
 import com.hous.hous_aos.ui.newrules.DayData
 import com.hous.hous_aos.ui.newrules.NewRulesUiState
 import com.hous.hous_aos.ui.newrules.component.State
 import javax.inject.Inject
 
 class NewRulesRepositoryImpl @Inject constructor(
-    private val remoteNewRulesDataSource: RemoteNewRulesDataSource
+    private val remoteNewRulesDataSource: com.hous.data.source.remote.RemoteNewRulesDataSource
```

Comment:
아 상당히 어지러워졌네요 ㅋㅋㅋㅋㅋ

Reply:
```suggestion
    private val remoteNewRulesDataSource: RemoteNewRulesDataSource
```
@murjune 
요거 모듈로 옮기면서 생긴건데 그냥 앞에 지운 다음에 import 하면 됩니다~~
리팩하다가 이런거 보이면 지워주세요~~~
아 근데 이건 내꺼네~~~ 하~~